.github/workflows/release.yml

```diff
@@ -54,32 +54,50 @@ jobs:
       - uses: actions/upload-artifact@v1
         if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
         with:
-          name: ${{ runner.os }}-wheels
+          name: ${{ runner.os }}-${{ matrix.python-version }}-wheel
           path: wheelhouse
+  test-release-wheel:
+    name: Test release wheels
+    runs-on: ${{ matrix.os }}
+    needs: [release-wheel]
+    strategy:
+      matrix:
+        os: ['macos-latest', 'windows-latest', 'ubuntu-18.04']
+        python-version: ['3.5', '3.6', '3.7']
+    steps:
+      - uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - uses: actions/download-artifact@v1
+        if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
+        with:
+          name: ${{ runner.os }}-${{ matrix.python-version }}-wheel
+          path: ./wheel
+      - name: Test wheel
+        if: github.event_name == 'schedule' || github.event_name == 'release' || matrix.python-version == '3.5'
+        shell: bash
+        run: |
+          pip install tensorflow~=2.1.0
+          pip install wheel/tensorflow_addons-*.whl
+          python -c "import tensorflow_addons as tfa; print(tfa.activations.gelu(0.2))"
```

Comment:
Since gelu is a possible candidate for moving to TF core. Could we replace with another custom-op? Say [gather_tree](https://github.com/tensorflow/addons/blob/master/tensorflow_addons/seq2seq/beam_search_decoder.py#L34) As seq2seq is likely to stay in Addons for the foreseeable future.

Reply:
I used `lisht` because it's simpler to setup than `gather_tree`.

Reply:
Thanks, I'm good with that. Depending on results of #1156 lisht custom-op may be removed at some point so lets just keep that in mind as those results get reported.